### DISCUSSION TITLE
SimpleLogin should allow fallback MX servers

### DISCUSSION THREAD

**Discussion by jmsgomes** [simple-login/app#769] [Feature Requests]
_[Filing this feature request per email reply from **SimpleLogin Team** `<hi@simplelogin.io>`]_

_By the way, keep up the great work! I'm really excited about SimpleLogin and switching more of my domains to using it._

---
I recently got an email as follows:

> **Subject: Please update _foo_ DNS on SimpleLogin**
>
> We have detected that your domain _foo_ doesn't have the DNS MX records correctly set up.
>
> Please re-run the MX check on `https://app.simplelogin.io/dashboard/domains/`_fooId_`/dns` and update them if needed.
>
> Without the MX records correctly set up, emails sent to the _foo_'s aliases  aren't properly and reliably handled.

I checked the MX records, and in my case, I have

```
10 mx1.simplelogin.co.
20 mx2.simplelogin.co.
100 mx1.forwardemail.net.
200 mx2.forwardemail.net.
1000 mxa.mailgun.org.
1000 mxb.mailgun.org.
```

**My understanding is that there's nothing fundamentally wrong/incompatible with this sort of setup** - **it should be ok/allowable to have other MX records with lower priorities** (that is, with a higher number; these would only be used in case SimpleLogin MX servers don't respond) as long as the domain manager knows what they're doing here and why.

_Additional note: it should have been made clear in the warning email what are the possible or expected consequences for leaving the records. I still don't know if this is a case of "it is ok to ignore this warning if you know what you're doing", "SimpleLogin will not serve this domain", "your account will be suspended" or something else. Hopefully, it's the case of "it is ok to ignore this warning if you know what you're doing"_

**Comment by Vnzq4Qje3zzU** [comment]
I think this PR solves it?

https://github.com/simple-login/app/pull/814

**Reply by nguyenkims** [reply]
Yes that's true. We plan to announce this feature later this week.
